NEWS.md

```diff
@@ -73,8 +73,10 @@ This section lists changes that do not have deprecation warnings.
     longer present. Use `first(R)` and `last(R)` to obtain
     start/stop. ([#20974])
 
-  * The `Diagonal` type definition has changed from `Diagonal{T}` to
-    `Diagonal{T,V<:AbstractVector{T}}` ([#22718]).
+  * The `Diagonal`, `Bidiagonal` and `SymTridiagonal` type definitions have changed from
```

Comment:
assuming that merges first? I'm still not so sure about that densifying constructor over there